docker/community/.env.example

```diff
@@ -12,12 +12,13 @@ HOST_NAME=http://localhost
 # available values 'dev', 'test', 'production', 'ci', 'local'
 NODE_ENV=local
 MONGO_MAX_POOL_SIZE=200
-MONGO_MIN_POOL_SIZE=100
+MONGO_MIN_POOL_SIZE=75
+
 # MONGO USER
 MONGO_INITDB_ROOT_USERNAME=root
 # MONGO PASSWORD
 MONGO_INITDB_ROOT_PASSWORD=secret
-MONGO_URL=mongodb://root:secret@mongodb:27017/novu-db?authSource=admin
+MONGO_URL=MONGO_URL=mongodb://$MONGO_INITDB_ROOT_USERNAME:$MONGO_INITDB_ROOT_PASSWORD@mongodb:27017/novu-db?authSource=admin
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
❓ **question:** Is the extra `MONGO_URL=` here a typo?‏

Reply:
Yes, It is let me fix this